src/experimental-features.test.js

```diff
@@ -0,0 +1,52 @@
+import { ExperimentalFeatures } from './experimental-features';
+
+describe('experimental features', () => {
+  const experimentalFeatures = new ExperimentalFeatures(localStorage);
+  beforeEach(() => {
+    const experimentalFeaturesData = {
```

Comment:
why is inside beforeEach?

Reply:
I added this here because there is a afterEach that does the cleanup.

Reply:
If you do not use the "real" localStorage, you can prepare each test independently and you do not need to pay attention to the cleanup.